Pract.01/Task12.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+int main() {
+
+	cout << "Seconds: ";
+	long long inputSeconds;
+	cin >> inputSeconds;
+
+	int days = inputSeconds * 1.15741E-5;
```

Comment:
There is an easier way instead of using multiplication, try using division and %.